benchmarks/test_serialize.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+import ipyparallel as ipp
+
+
+client = ipp.Client(profile='asv')
+view = client.spanning_tree_view()
```

Comment:
looks like this file is unused and can maybe be removed? Same for ``saved_resulta.pkl` (`resulta` typo)